app/graphql/mutations/account/create.rb

```diff
@@ -0,0 +1,42 @@
+class Mutations::Account::Create < Mutations::Base
+  include PublicMutation
+  include RateLimitedMutation
+  include FancyMutation
+
+  description 'Create a new Kitsu account'
+
+  rate_limit do
+    limit 20, per: 1.hour
+    limit 20, per: 2.days
+  end
+
+  input do
+    argument :name, String,
+      required: true,
+      description: 'The name of the user'
+
+    argument :email, String,
+      required: true,
+      description: 'The email address to reset the password for'
+
+    argument :password, String,
+      required: true,
+      description: 'The password for the user'
+
+    argument :external_identity, Types::Input::Account::ExternalIdentity,
+      required: false,
+      description: 'An external identity to associate with the account on creation'
+  end
+  result Types::Account
+  errors Types::Errors::Validation
+
+  def resolve(input:)
+    AccountMutator.create(**input)
+  rescue ActiveRecord::RecordInvalid => e
+    validation_errors = Types::Errors::Validation.for_record(e.record, transform_path: ->(path) {
+      path = path.map { |p| p == 'password_digest' ? 'password' : p }
+      ['input', *path]
+    })
+    errors.push(*validation_errors)
```

Comment:
I would be lying if I said I was happy about this part, but it solves the `password_digest` name leaking out as well as the `input` prefix. I might add a shortcut for the prefix in future, it seems like it'll be quite common (most models shouldn't have the weird `password_digest` bullshit)

Reply:
couldn't we just override the `graphql_name` for that error so it can still be `password_digest` but just show as `password` instead of having to have the logic here?